app/views/home/index.de.html.slim

```diff
@@ -31,7 +31,7 @@ section.jobs
                   .text-muted.mb-3 = posting.organization_name
                   .job-description = strip_tags(posting.description).truncate(200)
       .col-12.text-center.mt-3
-        = link_to('Weitere Einsätze anzeigen…', ENV['SMARTJOBBOARD_PUBLIC_URL'], class: 'btn btn-link')
+        = link_to('Weitere Einsätze anzeigen…', job_postings_path, class: 'btn btn-link')
```

Comment:
translation missing?

Reply:
It's a `.de.html.slim` so all is already translated

Reply:
Aaah did not read filename closely it seems